manager/backend/models.py

```diff
@@ -1,20 +1,23 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
 import uuid
 
 def generate_uuid():
     return uuid.uuid4().hex
 
 class Author(models.Model):
-    id = models.CharField(primary_key=True, default=generate_uuid(), editable=False, unique=True, max_length=100)
+    id = models.CharField(primary_key=True, default=generate_uuid, max_length=100, unique=True)
+    token = models.CharField(default="1234", max_length=100)
```

Comment:
does this need to have a foreign key to the User table?

Reply:
Added